integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioSessionMigrIstioEnabled.java

```diff
@@ -0,0 +1,402 @@
+    createConfigMapAndVerify(configMapName, domainUid, domainNamespace, Collections.EMPTY_LIST);
+
+    // create the domain object
+    Domain domain = createIstioDomainResource(domainUid,
+        domainNamespace,
+        adminSecretName,
+        OCIR_SECRET_NAME,
+        encryptionSecretName,
+        replicaCount,
+        miiImage,
+        configMapName,
+        clusterName);
+
+    // create model in image domain
+    createDomainAndVerify(domain, domainNamespace);
+    String adminServerPodName = domainUid + "-admin-server";
+    logger.info("Check admin service {0} is created in namespace {1}",
+        adminServerPodName, domainNamespace);
+    checkServiceExists(adminServerPodName, domainNamespace);
+
+    // check admin server pod is ready
+    logger.info("Wait for admin server pod {0} to be ready in namespace {1}",
+        adminServerPodName, domainNamespace);
+    checkPodReady(adminServerPodName, domainUid, domainNamespace);
+
+    // check managed server services created
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Check managed service {0} is created in namespace {1}",
+          managedServerPrefix + i, domainNamespace);
+      checkServiceExists(managedServerPrefix + i, domainNamespace);
+    }
+
+    // check managed server pods are ready
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Wait for managed pod {0} to be ready in namespace {1}",
+          managedServerPrefix + i, domainNamespace);
+      checkPodReady(managedServerPrefix + i, domainUid, domainNamespace);
+    }
+
+    return 0;
+  }
+
+  private Map<String, String> processHttpRequest(String curlUrlPath,
+                                                 String headerOption) {
+    String[] httpAttrArray = {"sessioncreatetime", "sessionid", "primary", "secondary", "count"};
+    Map<String, String> httpAttrInfo = new HashMap<String, String>();
+
+    // build curl command
+    String curlCmd = buildCurlCommand(curlUrlPath, headerOption);
+    logger.info("==== Command to set HTTP request and get HTTP response {0} ", curlCmd);
+
+    // set HTTP request and get HTTP response
+    ExecResult execResult = assertDoesNotThrow(
+        () -> execCommand(domainNamespace, adminServerPodName,
+        null, true, "/bin/sh", "-c", curlCmd));
+    if (execResult.exitValue() == 0) {
+      logger.info("\n ===== HTTP response is \n " + execResult.stdout());
+      assertAll("Check that primary server name is not null or empty",
+          () -> assertNotNull(execResult.stdout(), "Primary server name shouldn’t be null"),
+          () -> assertFalse(execResult.stdout().isEmpty(), "Primary server name shouldn’t be  empty")
+      );
+
+      for (String httpAttrKey : httpAttrArray) {
+        String httpAttrValue = getHttpResponseAttribute(execResult.stdout(), httpAttrKey);
+        httpAttrInfo.put(httpAttrKey, httpAttrValue);
+      }
+    } else {
+      fail("==== Failed to process HTTP request " + execResult.stderr());
```

Comment:
Remove the string "===="  form the fail statement.

Reply:
removed